src/psyclone/domain/nemo/transformations/nemo_loop_fuse.py

```diff
@@ -174,35 +172,42 @@ def validate_written_array(var_info1, var_info2, loop_variable):
             inconsistent indices, e.g. a(i,j) and a(j,i).
 
         '''
-
+        # pylint: disable=too-many-locals
```

Comment:
The implementation looks good but I am unsure where is this tested :)

The CI says its covered but `src/psyclone/tests/domain/nemo/transformations/nemo_loop_trans_test.py` doesn't seem to go thought this?